### PR TITLE
Translation pl patch 1

### PR DESCRIPTION
**Pull request by Rudzik99**
Translated some sentences with the word "pool" and a few others.

### CODE REVIEW

File: src/assets/i18n/pl.json [hunk 1]
```diff
@@ -34,7 +34,7 @@
   "<a href=\"https://www.ixsystems.com/community/\" target=\"_blank\"> TrueNAS Forums</a> - Find answers from other users in the forums.": "",
   "<a href=\"https://www.ixsystems.com/support/\" target=\"_blank\"> TrueNAS Licensing</a> - Learn more about enterprise-grade support.": "",
   "<a href=\"https://www.truenas.com/docs/hub/\" target=\"_blank\"> TrueNAS Documentation Hub</a> - Read and contribute to the open-source documentation.": "",
-  "<b>Boot Pool Condition:</b> {condition}": "",
+  "<b>Boot Pool Condition:</b> {condition}": "<b>Kondycja woluminu Boot:</b> {condition}",
```
Comment (AlexKarpov98) [inline src/assets/i18n/pl.json:37]: <b>Kondycja puli Boot:</b> {condition}

File: src/assets/i18n/pl.json [hunk 2]
```diff
@@ -1398,10 +1398,10 @@
   "Export Recycle Bin": "",
   "Export ZFS snapshots as <a href=\"https://docs.microsoft.com/en-us/windows/desktop/vss/shadow-copies-and-shadow-copy-sets\" target=_blank>Shadow Copies</a> for VSS clients.": "",
   "Export dataset keys which can be used to unlock datasets for ": "",
-  "Export/Disconnect": "",
-  "Export/Disconnect Pool": "",
-  "Export/disconnect pool: '": "",
-  "Exporting Pool": "",
+  "Export/Disconnect": "Eksportuj/Odłącz",
+  "Export/Disconnect Pool": "Eksportuj/Odłącz pulę",
+  "Export/disconnect pool: '": "Eksportuj/Odłącz pulę: '",
```
Comment (AlexKarpov98) [inline src/assets/i18n/pl.json:1403]: I think `Odłącz` is not proper to `Disconnect`. how about `Eksportuj/Rozłącz`?

File: src/assets/i18n/pl.json [hunk 3]
```diff
@@ -2658,31 +2658,31 @@
...
-  "Roll back ": "",
-  "Roll back snapshots": "",
-  "Roll back snapshots of ix_volumes": "",
-  "Rollback": "",
-  "Rolling back...": "",
+  "Restrict PAM": "Ogranicz PAM",
+  "Restrict share visibility to users with read or write access to the share. See the <a href=\"https://www.freebsd.org/cgi/man.cgi?query=smb.conf\" target=_blank>smb.conf</a> manual page.": "Ogranicz widoczność udziału do użytkowników z dostępem do odczytu lub zapisu. Zobacz stronę podręcznika <a href=\"https://www.freebsd.org/cgi/man.cgi?query=smb.conf\" target=_blank>smb.conf</a>.",
+  "Restricted": "Ograniczony",
+  "Retrieving Key": "Odzyskiwanie klucza",
+  "Retrieving catalog": "Pobieranie katalogu",
```
Comment (AlexKarpov98) [inline src/assets/i18n/pl.json:2661]: `Pobieranie` looks not good for me.

File: src/assets/i18n/pl.json [hunk 4]
```diff
@@ -2658,31 +2658,31 @@
...
+  "Revoked: ": "Unieważniony: ",
+  "Revoking Certificate": "Odwołanie certyfikatu",
+  "Revoking this CA will revoke the complete CA chain. This is a one way action and cannot be reversed. Are you sure you want to revoke this CA?": "Odwołanie tego urzędu certyfikacji (CA) spowoduje unieważnienie całego łańcucha zaufania. Jest to działanie jednokierunkowe i nie można go cofnąć. Czy na pewno chcesz unieważnić ten urząd certyfikacji?",
+  "Roll Back": "Cofnij",
+  "Roll back": "Cofnij",
+  "Roll back ": "Cofnij ",
+  "Roll back snapshots": "Cofnij migawki",
+  "Roll back snapshots of ix_volumes": "Cofnij migawki ix_volumes",
+  "Rollback": "Wycofanie",
+  "Rolling back...": "Wycofywanie...",
```
Comment (AlexKarpov98) [inline src/assets/i18n/pl.json:2681]: Rollback -> Wycofaj

**Review by AlexKarpov98 — CHANGES_REQUESTED** [review]
Thanks! LGTM

**Comment by undsoft** [comment]
@AlexKarpov98 Need to do something with this PR. Either retarget and fix it yourself, or close it.

**Review by AlexKarpov98 — APPROVED** [review]
LGTM!

**Comment by undsoft** [comment]
@AlexKarpov98 
PR was targeted to `master-old` (now stable/angelfish). Need to backport changes now.

**Pull request merged**